cli/build/main.rs

```diff
@@ -174,10 +173,13 @@ fn do_main() -> Result<(), Error> {
 		parity_wasm::serialize_to_file(save_raw_path, module.clone()).map_err(Error::Encoding)?;
 	}
 
-	parity_wasm::serialize_to_file(
-		&path,
-		ctor_module.expect("ctor_module can't be None, because 'constructor' argument is set to true in build"),
-	).map_err(Error::Encoding)?;
+	if let Some(ctor_module) = ctor_module {
```

Comment:
Just a grumble -- this interface looks a little bit confusing to me: if no deploy symbol is defined, and we run `wasm-build` without `--save-raw`, then no file will be written? Maybe we can return a warning in the above case or just change it so that "no deploy symbol" is equivalent to `--save-raw`?

Reply:
Indeed, it worked like you say before the #97